pom.xml

```diff
@@ -232,6 +232,27 @@
 				</configuration>
 			</plugin>
 			<plugin>
+				<groupId>org.apache.maven.plugins</groupId>
+				<artifactId>maven-checkstyle-plugin</artifactId>
+				<version>2.17</version>
+				<executions>
+					<execution>
+						<id>validate</id>
+						<phase>verify</phase>
+						<configuration>
+							<configLocation>checkstyle.xml</configLocation>
+							<encoding>UTF-8</encoding>
+							<consoleOutput>true</consoleOutput>
+							<failsOnError>true</failsOnError>
+							<linkXRef>false</linkXRef>
+						</configuration>
+						<goals>
+							<goal>check</goal>
```

Comment:
I don't have especially strong feelings about this, but `check` does appear to be the start goal from what I can tell from looking at the checkstyle plugin docs https://maven.apache.org/plugins/maven-checkstyle-plugin/usage.html and from other examples I've seen when googling.